docs-src/concepts/what-is-worker-versioning.md

```diff
@@ -0,0 +1,187 @@
+---
+id: what-is-worker-versioning
+title: What is Worker Versioning?
+sidebar_label: Worker Versioning
+description: Worker Versioning allows you to more easily deploy changes to Workflow Definitions.
+tags:
+  - explanation
+  - versioning
+---
+
+Worker Versioning simplifies the process of deploying changes to [WorkflowDefinitions](/workflows/#workflow-definition). We recommend that you read about Workflow Definitions
+before proceeding, because Workflow Versioning is largely concerned with helping to manage
+nondeterministic changes to those definitions.
+
+This feature can help you manage nondeterministic changes by providing a convenient way to ensure
+that [Workers](/workers) with different Workflow & Activity Definitions operating on the same [TaskQueue](/tasks/#task-queue) do not attempt to process [Workflow Tasks](/tasks/#workflow-task) and
```

Comment:
nit: the line here is much longer than the others.

Reply:
I think github just is lying there. They're all wrapped @ 100

Reply:
Oh, nevermind, whatever auto-formatter we have just ruins everthing. That's just the way it is.